reporters/reporter-default/src/main/java/org/radargun/reporting/csv/CsvReporter.java

```diff
@@ -287,4 +325,95 @@ private double round(double number, int places) {
       return bigDecimal.doubleValue();
    }
 
+   /**
+    * Comparator for sorting rows based on column value
+    * 
+    * @author zhostasa
```

Comment:
I just went over that discussion...it seems far from "decided"...
Anyway, I do not really care one way or the other